modules/ROOT/pages/api-search.adoc

```diff
@@ -0,0 +1,38 @@
+= Search Inside Your API Specification Dependencies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Because Anypoint Studio 7.4 enables you to import your API specifications from Exchange or Maven as dependencies of your project, those dependencies no longer locate as resources inside the `src/resources/api/` directory, but as ZIP file dependencies in your project's classpath. This change makes your API dependencies not searchable through the File Search option in Studio. To enable you to search for specific text inside your API dependencies, Studio introduces API Search.
+
+You can access the API Search menu from the taskbar at the top of the Anypoint Studio display, selecting *Search* > *Search...*, or by using the `control + H` keyboard shortcut.
+
+image::api-search.png[]
+
+. Type the text that you want to search inside your API dependencies. +
+* Use `\*` to search for any string. For example, `*-api-*` searches for strings containing `-api-`.
```

Comment:
GLOBALLY limit list items and steps to one sentence or fragment, moving related info directly below:
```suggestion
* Use `\*` to search for any string. 
For example, `*-api-*` searches for strings containing `-api-`.
```